edx-olive/configuration-old
# Upgrade to MySQL 5.7 and replace mongo version

Upgrade to MySQL 5.7 and replace Mongo DB version as MongoDB dismantled their download server, hence we have no source of download. Also, the production systems using mongo 3+ as well, so it has no point to run on mongodb 2.6+ in the devstack.

**Discussions**: N/A

**Dependencies**: None

**Sandbox URL**: N/A

**Merge deadline**: ASAP

**Testing instructions**:

1. Checkout the branch used for https://github.com/edx-olive/devstack/pull/1 and setup the devstack
2. run `vagrant ssh && mysql --version`
3. Check the mysql version to be `5.7`

**Author notes and concerns**:

If the https://github.com/edx-olive/devstack/pull/1 is already merged, you don't need to test anything as it is already tested; this branch is merged in `gabor/fix-box-creation` as it was needed to create the new base box image and the `gabor/fix-box-creation` branch is used by the devstack too.

**Reviewers**
- [ ] @jvdm
